src/js/main.js

```diff
@@ -1,3 +1,161 @@
-import { contacts } from '../data/contacts.js';
+import { contacts } from "../data/contacts.js";
 
-console.log(contacts);
+// console.log(contacts);
+
+// Initially 'contacts' is an object. Get an array of Contacts
+const contactsArray = contacts.results;
+
+// Phone and sell numbers not normalized. We need to do it and get a new array
+function getContactsArrayWithPhonesLikeANumber(contactsArray) {
+  return contactsArray.map((element) => {
+    const phone = element.phone.trim();
+    element.phone = normalizePhoneLikeANumber(phone);
+    const cell = element.cell.trim();
+    element.cell = normalizePhoneLikeANumber(cell);
+    // console.log(element);
+    return element;
+  });
+}
+
+// Convert an initial phone or cell number to a String Like Number
+function normalizePhoneLikeANumber(text) {
+  let phoneLikeANumber = "";
+  for (const char of text) {
+    if (
+      char !== "(" &&
+      char !== ")" &&
+      char !== "-" &&
+      char !== "+" &&
+      char !== " "
+    ) {
+      phoneLikeANumber += char;
+    }
+  }
+  return phoneLikeANumber;
+}
+
+// Get a new array of contacts with normalized phone and sell numbers and assign it to newContactsArray
+// We will use this array in our work further
```

Comment:
Really cool feature!
Its great use assuming you have the extra time and have this changes approved.
(for future assignemnts please make another branch where you can freely add features outside the scope of the task)